tsnet/example/tsnet-funnel/tsnet-funnel.go

```diff
@@ -34,6 +34,8 @@ func main() {
 	}
 	defer ln.Close()
 
+	fmt.Printf("Listening on https://%v\n", s.CertDomains()[0])
```

Comment:
Won't this crash in a common case?

Reply:
No, ListenFunnel does all the necessary checks